Creation of the Games folder and adding first game (#120)
KSP-KOS/KSLib: games/README.md

## KS-lib/games

> This is a public repository for games written in kerboscript

Author: made recommended change

> ### basic layout

Reviewer: ```suggestion
### Basic Layout
```

Author: made recommended change

> Each has it's own sub folder for ease of organization.

Reviewer: ```suggestion
Each game has its own sub folder for ease of organization.
```

Author: made recommended change

> With in each folder there will be a `game.ks` file that you can `RUN` to play the game and a README written using ascii text, or Github markdown files (.md) that contains some basic info about the game

Reviewer: ```suggestion
Within each folder there will be a `game.ks` file that you can `RUN` to play the game and a README written using ascii text, or Github markdown files (.md) that contains some basic info about the game.
```

Author: made recommended change

> Additionally there may also be needed libraries for the game in that case they can ether be found in the game folder or linked to in the README file.

Reviewer: ```suggestion
Additionally there may also be needed libraries for the game, which can either be placed in the game folder or linked to in the README file.
```

Author: made recommended change

## KS-lib/games

This is a public repository for games written in kerboscript.

### Basic layout

Each game has its own sub folder for ease of organization.
Within each folder there will be a `game.ks` file that you can `RUN` to play the game and a README written using ascii text, or Github markdown files (.md) that contains some basic info about the game.

> Additionally there may also be needed libraries for the game, which can either ether be placed in the game folder or linked to in the README file for the game.

Reviewer: either - ether

Author: made recommended change
